src/GitHunter.Desktop.WindowsForm/DesktopWindowsFormModule.cs

```diff
@@ -1,11 +1,26 @@
 ﻿using GitHunter.Application;
-using GitHunter.Desktop.Shared;
+using GitHunter.Core.DependencyProcesses;
+using GitHunter.Core.Modules;
 using Volo.Abp.Modularity;
 
-namespace GitHunter.Desktop.WindowsForm;
+namespace GitHunter.Desktop;
```

Comment:
Why was the namespace change to this? GitHunter.Desktop.Shared and GitHunter.Desktop.WindowsForm are same namespace now.

Reply:
I thought it would be better to use the same namespace of the module itself and the shared module